W-14222920-ping-cloud-duke (#214)
mulesoft/docs-code-builder: modules/ROOT/pages/_partials/acb-reusable-steps.adoc

* From the cloud IDE, click the menu icon, and select *View* > *Command Palette*.
====
// end::open-command-palette[]
//
//
// tag::check-connection-http[]
. Before starting your Mule app, confirm that the connection to the listener is successful: 
+
From the configuration XML, above the HTTP Listener configuration (`<<http:listener-config/>`), click *Test Connection*.

> * If the test succeeds, *Connection is valid* appears in a popup message.

Reviewer: ```suggestion
* If the test succeeds, you see a *Connection is valid* message.
```

Reviewer: Style guide says to omit "popup" and just refer to the element (message)

* EU Cloud (EU host): https://eu1.anypoint.mulesoft.com/exchange/[Exchange (EU)^]
====
// end::exchange-urls[]
//
//
// tag::no-project-name-reuse[]
// Cannot reuse project name

> To reuse an existing project name, you must delete the project first. See

Reviewer: ```suggestion
To reuse an existing project name, you must delete the project that is already using the name first. See
```

+
For more information, see xref:des-create-api-specs.adoc#review-spec-console[Review Your Spec in the API Console].
// end::after-add-fragment[]
//
//
// tag::in-progress[]
// In Progress
This topic is in progress.
// end::in-progress[]
//
//
// tag::proj-directory-warn[]
// Project Directory warning
Don't create the project within another project directory.
// end::proj-directory-warn[]
//

//
// tag::deploy-locally[]
. Click the image:icon-start-debug.png["",15,15] (*Start Debugging (F5)*) icon for *Debug Mule Application*.
+
Alternatively, if the *Start Debugging (F5)* icon is not present:

.. Click *Run and Debug*. 
.. If a *Select debugger* dropdown menu appears, select *Mule Xml Debugger* to initiate the debugging session.
+
image::anypoint-code-builder::run-debug-imported-folder.png["Run and Debug button and Mule XML Debugger menu item"]
+

> Make sure that the application deploys successfully with the `DEPLOYED` log message in the output panel:

Reviewer: ```suggestion
When the application deploys successfully, you see a `DEPLOYED` log message in the output panel:
```

+
[souce,output panel]
----
**********************************************************************
*              - - + DOMAIN + - -               * - - + STATUS + - - *
**********************************************************************
* default                                       * DEPLOYED           *
**********************************************************************
----
// end::deploy-locally[]
//

//
// tag::open-terminal[]
. From your IDE, open the Terminal window in the console.

* From the desktop IDE: Navigate to *View* > *Terminal*.

> * From the cloud IDE: Navigate to the menu icon > *Terminal* > *New Terminal*.

Reviewer: Do you have a menu icon, or we just assume everyone knows what a menu icon looks like?

Author: I don't have an icon. It's that hamburger menu. I think we just call it the menu icon. Do we provide a graphic for it usually?

Author: I have wondered if we should though... ¯\_(ツ)_/¯
